core/deployment/src/main/java/io/quarkus/deployment/builditem/nativeimage/JPMSExportBuildItem.java

```diff
@@ -23,6 +23,25 @@ public JPMSExportBuildItem(String moduleName, String packageName, GraalVM.Versio
         this(moduleName, packageName, exportSince, null);
     }
 
+    /**
+     * Creates a build item that indicates that a Java package should be exported for a specific GraalVM version range.
+     *
+     * @param moduleName the module name
+     * @param packageName the package name
+     * @param exportSince the version of GraalVM since which the package should be exported (inclusive)
+     * @param exportBefore the version of GraalVM before which the package should be exported (exclusive)
+     * @deprecated use {@link #JPMSExportBuildItem(String, String, GraalVM.Version, GraalVM.Version)} instead
+     */
+    @Deprecated
```

Comment:
Note, this is a bit confusing as it appears like I am adding a `Deprecated` method, but I actually changed the import and deprecated the method with the old types.